src/components/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```

Comment:
- [ ] When i tried to divide a number by `zero` am getting this error
![e](https://user-images.githubusercontent.com/44745855/129572908-25869871-98d6-4f7a-9377-2e9d3be85da3.png)

Normally it has to display an `error` in the screen kindly fix that before merging

Reply:
:ok:  I didn't realize about that possibility. Thanks for noticing, I got it fixed already :heavy_check_mark: